libs/core/src/lib/token/token.component.html

```diff
@@ -1,4 +1,4 @@
-<span
+<span #spanElement
```

Comment:
#spanElement is not the best naming for this considering the fact there are another 2 spans :)
The rest looks gook!

Reply:
Why did you decided to base on `span` reference? `ElementRef` was not enough?

Reply:
@salarenko I don't understand, what is your suggested alternative here?

Reply:
I'm just asking why did you switched from this solution?
![image](https://user-images.githubusercontent.com/17496353/88029919-e2183380-cb3a-11ea-9c44-3ec6c8afba6f.png)

Reply:
Because the value returned from that included the margin which we don't want for this calculation